tbx/project_styleguide/templates/patterns/pages/team/team_listing.yaml

```diff
@@ -1,27 +1,37 @@
 context:
   page:
-    title: This is us, <span>the people that make Torchbox</span>.
+    title: Team
+    strapline: This is us, <span>the people that make Torchbox</span>.
```

Comment:
The strapline mentioned above with a `span` tag.

Reply:
I will remove that `span` in the FE ticket